clients/cli/src/main/java/org/apache/gravitino/cli/commands/Command.java

```diff
@@ -73,11 +109,22 @@ protected GravitinoClient buildClient(String metalake) throws NoSuchMetalakeExce
    * @return A configured {@link GravitinoAdminClient} instance.
    */
   protected GravitinoAdminClient buildAdminClient() {
+    Builder<GravitinoAdminClient> client = GravitinoAdminClient.builder(url);
+
     if (ignoreVersions) {
-      return GravitinoAdminClient.builder(url).withVersionCheckDisabled().build();
-    } else {
-      return GravitinoAdminClient.builder(url).build();
+      client = client.withVersionCheckDisabled();
     }
+    if (authentication != null) {
+      if (authentication.equals("simple")) {
+        if (userName != null && !userName.isEmpty()) {
+          client = client.withSimpleAuth(userName);
+        } else {
+          client = client.withSimpleAuth();
+        }
+      }
```

Comment:
for the else part, we'd better output something to the log.

Reply:
I'm not sure whywe woudl only log the else part as there two types of simple authentication one with a user name and one with no user name.

Reply:
In general the CLI only outputs to standard output or error - currently it doesn't log anything

Reply:
If user typed a wrong value for the auth parameter, and then the CLI just ignore that, it will lead to unexpected behavior, which will be difficult for the user to debug. That is my point: when an unrecognized input received, should we just ignore that, or give some error or warning?

Reply:
> for the else part, we'd better output something to the log.

I mean if the "auth" is given but not be "simple". There is nothing happen now.

Reply:
Auth is either null or "simple" it can have no other values as it is set via a flag not passed as an argument. Any other unrecognized user input is caught well before this, so if I did add an else the code in it would never be called.

Reply:
I see, thanks for the information.

Reply:
However, it would be possible be incorrect in the config file or enviroment variable, I'll add some output just in case.